contrib/config/vault/docker/README.md

```diff
@@ -46,16 +46,16 @@ Alternative Steps using `bound_cidr_list` (see [Using Hashicorp Vault CIDR List
 
 ## Steps
 
-This configures an app role that requires log in with `role-id` and `secret-id` to login.  This is the default role setting where `bind_seccret_id` is enabled.
+This configures an app role that requires log in with `role-id` and `secret-id` to login.  This is the default role setting where `bind_secret_id` is enabled.
 
 ### Step 1: Configure Dgraph and Vault Versions
 
 ```bash
-export DGRAPH_VERSION="v21.03"  # default is 'latest'
-export VAULT_VERSION="1.7.0"    # default is 'latest'
+export DGRAPH_VERSION="v21.03.0" # default is 'latest'
```

Comment:
should we update these to v23 now?

Reply:
These are minor fixes that I hope to get in.  I could test the new version, but this could take time. I should probably test with a new version of Vault as well.